cosmwasm/utils.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const { readFileSync } = require('fs');
+const { calculateFee, GasPrice } = require('@cosmjs/stargate');
+const { instantiate2Address } = require('@cosmjs/cosmwasm-stargate');
+const { getSaltFromKey } = require('../evm/utils');
+
+const pascalToSnake = (str) => {
+    return str.replace(/([A-Z])/g, (group) => `_${group.toLowerCase()}`).replace(/^_/, '');
+};
+
+const uploadContract = async (config, options, wallet, client) => {
+    const [account] = await wallet.getAccounts();
+
+    const wasm = readFileSync(`${options.artifactPath}/${pascalToSnake(options.contractName)}${options.aarch64 ? '-aarch64' : ''}.wasm`);
+
+    const gasPrice = GasPrice.fromString(`0.00005u${config.axelar.tokenSymbol.toLowerCase()}`);
+    const uploadFee = calculateFee(5000000, gasPrice);
```

Comment:
Should this value really be hardcoded?

Reply:
good catch, changed it to read from the config